### PR TITLE
Added Series.ewm_std/2 and Series.ewm_var/2

### PR DESCRIPTION
**Pull request by rtvu**
Closes [#776](https://github.com/elixir-explorer/explorer/issues/776).

### CODE REVIEW

File: lib/explorer/backend/lazy_series.ex [hunk 1]
```diff
@@ -73,6 +73,8 @@ defmodule Explorer.Backend.LazySeries do
     window_sum: 5,
     window_standard_deviation: 5,
     ewm_mean: 5,
+    ewm_std: 6,
+    ewm_var: 6,
```
Comment (josevalim) [inline lib/explorer/backend/lazy_series.ex:77]: We call these things `variance` and `standard_deviation` below. Should we rename these two to match them?

**Review by josevalim — COMMENTED** [review]
Looks great to me, I have added only one suggestion to the name (which can likely be addressed with a search+replace).

File: lib/explorer/backend/lazy_series.ex (hunk 1 above)
Comment (rtvu) [reply lib/explorer/backend/lazy_series.ex:77]: The function names were copied as is from `polars`. We should rename to match the convention used in `Explorer`.

**Comment by rtvu** [comment]
The functions have been renamed to `ewm_standard_deviation/2` and `ewm_variance/2`.

File: lib/explorer/polars_backend/series.ex [hunk 2]
```diff
@@ -533,6 +533,16 @@ defmodule Explorer.PolarsBackend.Series do
     Shared.apply_series(series, :s_ewm_mean, [alpha, adjust, min_periods, ignore_nils])
   end
 
+  @impl true
+  def ewm_standard_deviation(series, alpha, adjust, bias, min_periods, ignore_nils) do
+    Shared.apply_series(series, :s_ewm_standard_deviation, [alpha, adjust, bias, min_periods, ignore_nils])
```
Comment (josevalim) [inline lib/explorer/polars_backend/series.ex:538]:
```suggestion
    Shared.apply_series(
      series,
      :s_ewm_standard_deviation,
      [alpha, adjust, bias, min_periods, ignore_nils]
    )
```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**